odoo/tests/common.py

```diff
@@ -283,8 +273,7 @@ def _request_handler(self, s: Session, r: PreparedRequest, /, **kw):
 
         _logger.getChild('requests').info(
             "Blocking un-mocked external HTTP request %s %s", r.method, r.url)
-        raise requests.exceptions.ConnectionError(
-            f"External requests verboten (was {r.method} {r.url})")
+        raise BlockedRequest(f"External requests verboten (was {r.method} {r.url})")
```

Comment:
It seems that whenever you have a post/get request in your code, this test fails now because of these changes. I guess it is not itentional